runDecamTest.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+print_error() {
+    >&2 echo $@
+}
+
+WORKSPACE=DECam
+mkdir -p ${WORKSPACE}
+if [ -d ${WORKSPACE} ]; then
+   rm -rf ${WORKSPACE}
+fi
+
+# cleanup and create directories
+echo "Ingesting Raw data"
+INPUT=${WORKSPACE}/input
+mkdir -p $INPUT
+OUTPUT=${WORKSPACE}/output
+mkdir -p $OUTPUT
+
+echo "lsst.obs.decam.DecamMapper" > ${INPUT}/_mapper
+
+RAWDATA=${VALIDATION_DATA_DECAM_DIR}
+ingestImagesDecam.py ${INPUT} ${RAWDATA}/instcal/*.fz --mode link
```

Comment:
`${RAWDATA}` needs to be quoted. Consider running `shellcheck` on the shell scripts.